vnext/local-cli/runWindows/utils/deploy.js

```diff
@@ -164,15 +164,25 @@ async function deployToDesktop(options, verbose) {
     verbose,
   );
 
+  const devmodeText = 'Enabling Developer Mode';
+  const devmodeEnable = `-ExecutionPolicy RemoteSigned Import-Module "${windowsStoreAppUtils}"; EnableDevmode "${script}"`;
```

Comment:
>`- [](start = 24, length = 2)

-NoProfile

Reply:
good to know

---
In reply to: [336322464](https://github.com/microsoft/react-native-windows/pull/3447#discussion_r336322464) [](ancestors = 336322464)